app/controllers/api/projects_controller.rb

```diff
@@ -52,6 +52,10 @@ def destroy
       head :ok
     end
 
+    def show_context
+      render :context, formats: [:json]
```

Comment:
maybe a comment explaining the purpose would be a good? as I don't think it's easy to understand why we're doing this just by reading the code...